### PR TITLE
Allow to use custom font

### PR DESCRIPTION
**Pull request by anvox**
Allow to use custom font with local font file provided. 

Changes: 

* Add options `font` to pass font config to RMagick Drawing.

### CODE REVIEW

File: lib/magic_cloud/canvas.rb [hunk 1]
```diff
@@ -56,7 +56,11 @@ def render
     private
 
     def set_text_options(draw, options)
-      draw.font_family = options[:font_family]
+      if options[:font_source]
+        draw.font = options[:font_source]
```
Comment (zverok) [inline lib/magic_cloud/canvas.rb]: Don't you think we should name that exactly like option in RMagick is named (e.g. "font")? As far as I understand, it is either family+weight+style, or path to font file (which contains only one weight and style, so setting them in this case is pointless).

**Review by zverok — COMMENTED** [review]
Hey. As I don't have much time/ability to maintain the gem, I can accept this PR "as is". Or you can take my comment into an account. Or, if by change you feel really enthusiastic, you may want to take a look at (stale currently) [refactoring branch](https://github.com/zverok/magic_cloud/commits/simplify) and probably you'd like to evolve it somehow?

File: lib/magic_cloud/canvas.rb (hunk 1 above)
Comment (anvox) [reply lib/magic_cloud/canvas.rb]: Thanks, I just updated option name.

**Comment by anvox** [comment]
Just warm up after a long time, it took me more time to recall the context and config than I expected.

**Comment by zverok** [comment]
Thanks!

**Pull request merged**